common/models/middleware.js

```diff
@@ -16,6 +16,23 @@ module.exports = function(Middleware) {
     }
     return phase;
   };
+  Middleware.getPhaseFromId = function(id) {
+    const parts = id.split('.');
+    if (parts.length && parts.length > 1) {
```

Comment:
I meant if id isn't a string it will throw an error so split should always return an array and arrays always have a length property